src/main/cli/settings.c

```diff
@@ -1152,11 +1150,11 @@ const clivalue_t valueTable[] = {
     { "ff_boost",                   VAR_UINT8 | PROFILE_VALUE,  .config.minmaxUnsigned = { 0, 50 }, PG_PID_PROFILE, offsetof(pidProfile_t, ff_boost) },
 
 #ifdef USE_DYN_IDLE
-    { "idle_min_rpm",               VAR_UINT8 | PROFILE_VALUE, .config.minmaxUnsigned = { 0, 100 }, PG_PID_PROFILE, offsetof(pidProfile_t, idle_min_rpm) },
-    { "idle_adjustment_speed",      VAR_UINT8 | PROFILE_VALUE, .config.minmaxUnsigned = { 25, 200 }, PG_PID_PROFILE, offsetof(pidProfile_t, idle_adjustment_speed) },
-    { "idle_p",                     VAR_UINT8 | PROFILE_VALUE, .config.minmaxUnsigned = { 10, 200 }, PG_PID_PROFILE, offsetof(pidProfile_t, idle_p) },
-    { "idle_pid_limit",             VAR_UINT8 | PROFILE_VALUE, .config.minmaxUnsigned = { 10, 255 }, PG_PID_PROFILE, offsetof(pidProfile_t, idle_pid_limit) },
-    { "idle_max_increase",          VAR_UINT8 | PROFILE_VALUE, .config.minmaxUnsigned = { 0, 255 }, PG_PID_PROFILE, offsetof(pidProfile_t, idle_max_increase) },
+    { "idle_min_rpm",           VAR_UINT8 | PROFILE_VALUE, .config.minmaxUnsigned = { 0, 100 }, PG_PID_PROFILE, offsetof(pidProfile_t, idle_min_rpm) },
+    { "dyn_idle_p_gain",            VAR_UINT8 | PROFILE_VALUE, .config.minmaxUnsigned = { 1, 250 }, PG_PID_PROFILE, offsetof(pidProfile_t, dyn_idle_p_gain) },
+    { "dyn_idle_i_gain",            VAR_UINT8 | PROFILE_VALUE, .config.minmaxUnsigned = { 1, 250 }, PG_PID_PROFILE, offsetof(pidProfile_t, dyn_idle_i_gain) },
+    { "dyn_idle_d_gain",            VAR_UINT8 | PROFILE_VALUE, .config.minmaxUnsigned = { 0, 250 }, PG_PID_PROFILE, offsetof(pidProfile_t, dyn_idle_d_gain) },
+    { "dyn_idle_max_increase",      VAR_UINT8 | PROFILE_VALUE, .config.minmaxUnsigned = { 10, 250 }, PG_PID_PROFILE, offsetof(pidProfile_t, dyn_idle_max_increase) },
```

Comment:
Why was this renamed - this just creates confusion and extra work. Also, why was the upper limit changed?

Reply:
The previous `idle_p` was not in fact a P gain, it was overall integrator gain.  

The previous `idle_adjustment_speed` wasn't a speed, and there was no equivalent for it in the new PID based code.

The defaults for each value of the PIDs are now different.  Having different names in 4.3 avoids problems with cut and paste of inappropriate defaults from 4.2.

In general I felt that the names of all parameters related to dynamic idle should have the form `dyn_idle_xxx` to distinguish them for parameters unrelated to dynamic idle, such as DShot idle.

This makes it much easier to locate all dynamic idle related parameters in the code and for users when they search in CLI.

Other than that `idle_min_rpm` was used in the CMS, and I didn't want to make a change to a CMS value at this time,  I would have renamed it also, since it is only applicable when dynamic idle is active, and could otherwise be confused with DShot idle.  I think it would be best to re-name it also, to `dyn_idle_min_rpm`.

Reply:
The upper limit was changed to permit up to 2.5x greater PID gains.
I'm not entirely sure how much greater the absolute upper limit of PID gain might be.  
The intent is that 100 = 'normal', so that 250 means 2.5x normal, and 25 means 0.25x normal.

Reply:
I've set the defaults back to 50 to allow up to 5x greater values on the upside, and down to off on the downside.
The wide range is for testing and development, most likely will be narrowed once weget more experience with suitability of defaults.

Reply:
Any reason to not allow these values to go to 255 like before?

Reply:
good point, none